test/org/jetbrains/plugins/scala/codeInsight/intentions/booleans/ReplaceEqualsOrEqualityInMethodCallExprIntentionTest.scala

```diff
@@ -21,8 +21,34 @@ class ReplaceEqualsOrEqualityInMethodCallExprIntentionTest extends ScalaIntentio
 
   def testReplaceQuality2() {
     val text = "if (a.eq<caret>uals(false)) return"
-    val resultText = "if (a.<caret>==(false)) return"
+    val resultText = "if (a <caret>== false) return"
 
     doTest(text, resultText)
   }
+
+  def testReplaceQuality2_withSpace() {
+    val text = "if (a.eq<caret>uals( false  )) return"
+    val resultText = "if (a <caret>== false) return"
+
+    doTest(text, resultText)
+  }
+
+  def testReplaceTuple(): Unit = {
+    (true, false).equals(false, false)
+    val text = "(true, false).equ<caret>als  (false, false)"
+    //TODO desired (true, false) <caret>== (false, false)
+    val resultText = "(true, false) <caret>==(false, false)"
+
+    doTest(text, resultText)
+  }
+
+  def testReplaceTuple_2(): Unit = {
+    (true, false).equals(false, false)
+    val text = "if((true, false).equ<caret>als  (false, false)) return "
+    //TODO desired (true, false) <caret>== (false, false)
+    val resultText = "if ((true, false) <caret>==(false, false)) return"
```

Comment:
I can't seem to get the space inserted after `==` and before `(`.

When I getText of the psi element resulting from the expression listed in line 118 and [line 119](https://github.com/neowulf33/intellij-scala/blob/425912dbc9099ede89291ae62ad70b6953c62398/src/org/jetbrains/plugins/scala/codeInsight/intention/booleans/ReplaceEqualsOrEqualityInMethodCallExprIntention.scala#L119), the desired space exists.

Does the space get removed when the following line gets executed?
```
PsiDocumentManager.getInstance(project).commitDocument(editor.getDocument)
```